tests/Foundation/FoundationAliasLoaderTest.php

```diff
@@ -52,6 +60,8 @@ public function testLoaderCanAliasAndLoadClasses()
 
     public function testSetAlias()
     {
+        AliasLoader::setInstance(null);
```

Comment:
It’s needed because the tests are failing in CI because the alias instance is a singleton and for some reason on CI more aliases are added locally it does not happen. Look at first few builds